cmd/arcaflow/main.go

```diff
@@ -57,55 +58,33 @@ func main() {
 	workflowFile := "workflow.yaml"
 	printVersion := false
 
-	flag.BoolVar(&printVersion, "version", printVersion, "Print Arcaflow Engine version and exit.")
-	flag.StringVar(
-		&configFile,
-		"config",
-		configFile,
-		"The Arcaflow configuration file to load, if any.",
+	const (
+		helpUsage    = "Display this help output."
+		versionUsage = "Print Arcaflow Engine version and exit."
+		configUsage  = "The Arcaflow configuration file to load, if any."
+		inputUsage   = "The workflow input file to load. May be outside the workflow " +
+			"directory."
+		contextUsage  = "The workflow directory to run from."
+		workflowUsage = "The workflow file in the current directory to load."
```

Comment:
A little experimentation confirms your findings:  the `-input` and `-config` options are required while the `-context` and `-workflow` options have useful defaults (the current directory and `workflow.yaml`, respectively).

So, assuming that we want to make the help text match the existing behavior,
- the correction here to the the input usage is warranted;
- the config usage should not say "if any" -- there _has_ to be one (e.g., `/dev/null` is rejected); and,
- the changes to the usage messages for the context and the workflow should presumably have the explanation of the default restored.

Reply:
I'm working on adding a default config as the way of addressing the "if any" statement. I think this simplifies the input and seems to model what the original expected behavior was according to that help language.

The explanation of the defaults in the help text is redundant when using the `flag` module since it will automatically display any defaults.